src/nls/root/strings.json

```diff
@@ -39,6 +39,7 @@
         "SET_TYPE_TRACKING": "Set Type Tracking",
         "SET_TYPE_ALIGNMENT": "Set Type Alignment",
         "UNGROUP_LAYERS": "Ungroup layers",
+        "REORDER_LAYERS": "Reorder Layers",
```

Comment:
I've been trying to keep meaningful history state "names" when possible ... in preparation for some day displaying them to the user?  Or at least for troubleshooting.